thirdparty/kpvcrlib/CMakeLists.txt

```diff
@@ -20,15 +20,17 @@ set(ANTIWORD_INCLUDE_DIR ${CR_3RDPARTY_DIR}/antiword)
 set(CHM_INCLUDE_DIRS ${CR_3RDPARTY_DIR}/chmlib/src)
 set(CREGINE_INCLUDE_DIR ${CRE_DIR}/include)
 
+assert_var_defined(JCONFIG_INCLUDE_DIR)
```

Comment:
Any reason for you to move this at top ? Looks a bit related to JPEGLIB - or may be not, dunno :)

Reply:
It was to keep the same order as the order in which they're defined/used below ;).